lib/src/components/NxSmallThreatCounter/__tests__/NxSmallThreatCounter.test.tsx

```diff
@@ -4,280 +4,157 @@
-    const noneCounter = component.find('.nx-small-threat-counter--none');
-    expect(noneCounter.find('.nx-small-threat-counter__count')).toHaveText('0');
-    expect(noneCounter.find('.nx-small-threat-counter__category')).toHaveText('None');
-  });
+  it('renders a counter with an accessible name of Critical when criticalCount is specified', async function() {
+    const component = quickRender({criticalCount: 0}),
+        counter = getCounter(component, 'critical');
 
-  it('renders an .nx-small-threat-counter--unspecified when unspecifiedCount is specified', function() {
-    const component = getMountedComponent({unspecifiedCount: 0});
-    const container = component.find('.nx-small-threat-counter-container');
-    expect(container).toExist();
-    const unspecifiedCounter = component.find('.nx-small-threat-counter--unspecified');
-    expect(unspecifiedCounter.find('.nx-small-threat-counter__count')).toHaveText('0');
-    expect(unspecifiedCounter.find('.nx-small-threat-counter__category')).toHaveText('Unspecified');
-  });
-
-  it('renders all six indicators', function() {
-    const component = getMountedComponent({
-      criticalCount: 66,
-      severeCount: 55,
-      moderateCount: 44,
-      lowCount: 33,
-      noneCount: 22,
-      unspecifiedCount: 42
-    });
-    const criticalCount = component.find('.nx-small-threat-counter--critical'),
-        severeCount = component.find('.nx-small-threat-counter--severe'),
-        moderateCount = component.find('.nx-small-threat-counter--moderate'),
-        lowCount = component.find('.nx-small-threat-counter--low'),
-        noneCount = component.find('.nx-small-threat-counter--none'),
-        unspecifiedCount = component.find('.nx-small-threat-counter--unspecified');
-    expect(criticalCount.find('.nx-small-threat-counter__count')).toHaveText('66');
-    expect(criticalCount.find('.nx-small-threat-counter__category')).toHaveText('Critical');
-    expect(severeCount.find('.nx-small-threat-counter__count')).toHaveText('55');
-    expect(severeCount.find('.nx-small-threat-counter__category')).toHaveText('Severe');
-    expect(moderateCount.find('.nx-small-threat-counter__count')).toHaveText('44');
-    expect(moderateCount.find('.nx-small-threat-counter__category')).toHaveText('Moderate');
-    expect(lowCount.find('.nx-small-threat-counter__count')).toHaveText('33');
-    expect(lowCount.find('.nx-small-threat-counter__category')).toHaveText('Low');
-    expect(noneCount.find('.nx-small-threat-counter__count')).toHaveText('22');
-    expect(noneCount.find('.nx-small-threat-counter__category')).toHaveText('None');
-    expect(unspecifiedCount.find('.nx-small-threat-counter__count')).toHaveText('42');
-    expect(unspecifiedCount.find('.nx-small-threat-counter__category')).toHaveText('Unspecified');
+    expect(counter).toBeInTheDocument();
+    expect(counter).toHaveTextContent('0');
+    await waitFor(() => expect(counter).toHaveAccessibleName('Critical'));
```

Comment:
If you call `await runTimers()` before the expectation I don't think you'll need the `waitFor`

Reply:
Updated here 90384b82c1b3e088c4144729e663709de552c392